### ISSUE TITLE
Custom class (?)

### ISSUE THREAD

**Issue by jonocarroll** [mpadge/stellar#18]
Should the returned object of `getstars` be of a custom class (e.g. `class = "stellar") with its own print method? Perhaps some other methods too (e.g. summary)?

**Comment by mpadge** [comment]
yeah, that's a solid idea, especially for the ease of a `print` method
